src/Engine/Utils.java

```diff
@@ -86,7 +87,7 @@ else if (includeTerrain && resident == null && // Peeps ain't there.
       {
         // Add any location that is empty and supports movement of the cargo unit.
         if( (map.isLocationEmpty(loc) || map.getLocation(loc).getResident() == transport)
-            && cargo.model.movePower >= cargo.model.propulsion.getMoveCost(map.getEnvironment(loc.xCoord, loc.yCoord)) )
+            && MoveType.IMPASSABLE > cargo.model.propulsion.getMoveCost(map.getEnvironment(loc.xCoord, loc.yCoord)) )
```

Comment:
Does this change... actually impact anything?

Reply:
You can now unload infantry onto snowy mountains. I double-checked AW2.

Reply:
Nice.